### PR TITLE
HW1-David Cheng

### PR DESCRIPTION
**Pull request by xiaorancheng**
I finished HW1 according to the process given by the "general workflow" on the course website. It was not difficult to use Github.  However, I encountered a few problems editing the R Markdown file. I attempted to input headers but I failed. I just saw the raw text rather than the header. Also, I didn't know how to start a new line. I searched on Google and followed the instructions on https://guides.github.com/features/mastering-markdown/. Finally, I managed to finish it.

I still don't know how to delete commits on Github. Maybe the commits can't be deleted. I can only delete a repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # hw01
-Repository for first CFSS homework assignment
+Repository for first CFSS homework assignment  
+
+## Introduction of Myself
```
Comment (jmarvelcoen) [inline README.md:4]: Good use of headers!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+
+Hi everyone!  
+I'm **Xiaoran Cheng** from China. You can call me **David**.
```
Comment (jmarvelcoen) [inline README.md:7]: Good use of bold!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+
+1. *Hometown*: Beijing
```
Comment (jmarvelcoen) [inline README.md:9]: Good use of lists!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,16 @@
...
+2. *Interest*:
+   * Soccer
+   * Reading
+
+You guys can follow me on [Facebook](https://www.facebook.com/profile.php?id=100005247065834)
```
Comment (jmarvelcoen) [inline README.md:14]: Good use of hyperlink!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,16 @@
...
+
+![This is me~](https://scontent.xx.fbcdn.net/v/t1.0-9/15977342_645541108964119_622605024512763586_n.jpg?oh=f7f0f31961a4ed3294bd152ea965b97e&oe=58D9D2A1)
```
Comment (jmarvelcoen) [inline README.md:16]: Good use of picture! I'm not sure you need the ~ at the end of the caption, though.

**Review by jmarvelcoen — COMMENTED** [review]
Review Summary:
Coding style: N/A (no coding)
Coding strategy: N/A (no coding)
Presentation: Excellent
Achievement/creativity: Good (might have been nice to have some details on what program you're in/research interests).
Ease of access: Excellent!

Overall, a good bio, including headers, boldface, italics, lists, hyperlinks, and an image. It might have been nice to have a little more biographical info, everything was put together very nicely, and everything opened and displayed properly. The only issue was that you have an extraneous tilde in the caption of the photo which could be removed.
It sounds like you had some issues with headers, based on your comment, but everything came out fine! I don't think you need to delete commits, as they serve as a record of the changes you made.

All in all, good job!

**Comment by jmausolf** [comment]
## Evaluation HW-01 for David Cheng

| Topic | Excellent | Satisfactory | Needs Work |
| --- | --- | --- | --- |
| **Coding style** |        x  |     |             |
| **Coding strategy** |     x   |        |        |
| **Presentation: graphs** |      |     |       |
| **Presentation: tables** |       |       |     |
| **Achievement, creativity** |x      |      |         |
| **Ease of access** | x |                 |             |

Remarks:
- For the header of the assignment, you probably could have removed the "hw01" header and subsequent lines with the assignment details. If you would like to include these, they could probably go in your pull request title/notes.
- Nice inclusion of a photo. Good job.
- Otherwise, you effectively used five of the five style elements, italics, photo, headers, list, and link. Good job.
- For git messages, try to be more descriptive with what changes you made, not simply `version 1` or `version 2`. What changes did you make? Write `embedded image for bio` or `fixed broken hyperlink`. 
- Good job overall.

**Pull request closed without merging**